SYCL/AtomicRef/add.h

```diff
@@ -281,100 +287,44 @@ template <access::address_space space, typename T, typename Difference = T,
 void add_test_scopes(queue q, size_t N) {
   std::vector<memory_scope> scopes =
       q.get_device().get_info<info::device::atomic_memory_scope_capabilities>();
-#if defined(SYSTEM)
-  if (std::find(scopes.begin(), scopes.end(), memory_scope::system) ==
+  if (std::find(scopes.begin(), scopes.end(), memory_scope::system) !=
       scopes.end()) {
-    std::cout << "Skipping test\n";
-    return;
+    add_test<space, T, Difference, order, memory_scope::system>(q, N);
   }
-  add_test<space, T, Difference, order, memory_scope::system>(q, N);
-#elif defined(WORK_GROUP)
-  if (std::find(scopes.begin(), scopes.end(), memory_scope::system) ==
+  if (std::find(scopes.begin(), scopes.end(), memory_scope::system) !=
```

Comment:
```suggestion
  if (std::find(scopes.begin(), scopes.end(), memory_scope::work_group) !=
```
Just noticed this is actually wrong.